### DISCUSSION TITLE
ECONNREFUSED 172.24.0.6:3001 (ECONNREFUSED) after BACKUP and RECOVERY

### DISCUSSION THREAD

**Discussion by Vendo232** [immich-app/immich#4544] [Q&A]
I have migrated immich to a new HW, followed Backup and Restore process, all went well, I can see photos but when I brows further the pictures are blury and they are not loading. here are pictures of the issue.

Immich is a great app, but I`m not profesional programmer and the Backup/Restore process currently is very volatile and intimidating 

![image](https://github.com/immich-app/immich/assets/44098230/f1e6e33c-b374-4a80-afa9-d797a3239daf)

![image](https://github.com/immich-app/immich/assets/44098230/2ffc0dff-c102-4574-be67-f44b52a2d045)

**Comment by alextran1502** [comment]
Can you share the server logs?

**Reply by Vendo232** [reply]
[immich_log_10192023.txt](https://github.com/immich-app/immich/files/13045607/immich_log_10192023.txt)
Here you go Alex, thank you for looking into this

**Reply by Vendo232** [reply]
I have tried to recreate all containers with latest container pull and all seems to be working now.

![image](https://github.com/immich-app/immich/assets/44098230/7c4bd04e-5eed-4dc2-b5a4-667c81470b24)

**Comment by jrasm91** [comment]
We might need to update the documentation. The backup and restore docs omit some post-backup steps that have to run. Basically, the restore process doesn't restore generated content. You either need to also have copied your `thumbs/` and `encoded-videos/` directories or run the thumbnail generation job for all (you can turn off ML jobs while this is running to avoid re-processing them again).

**Reply by Vendo232** [reply]
please also add this to the documentation, it saved me yesterday

before Restore when containers are created in Portainer /Stack mode containers autostart and create some database first thing. Running these commands helped me to wipe and start clean. 

```
 sudo docker exec -i immich_postgres dropdb immich -U postgres
 sudo docker exec -i immich_postgres createdb immich -U postgres
```
